[SIM] Logged memory size 0 at Mon Jun  2 23:54:39 2025
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 45, in <module>
[SIM] Logged memory size 0 at Mon Jun  2 23:57:15 2025
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 45, in <module>
    main()
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 42, in main
    time.sleep(SYNC_PAUSE)
KeyboardInterrupt
[SIM] Logged memory size 0 at Tue Jun  3 00:00:31 2025
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 45, in <module>
[SIM] Logged memory size 0 at Tue Jun  3 00:02:48 2025
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 45, in <module>
    main()
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 42, in main
    time.sleep(SYNC_PAUSE)
KeyboardInterrupt
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 5, in <module>
    from common.black_swan_agent.mutation_memory import load_memory, save_memory
ModuleNotFoundError: No module named 'common'
Traceback (most recent call last):
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 5, in <module>
    from common.black_swan_agent.mutation_memory import load_memory, save_memory
ModuleNotFoundError: No module named 'common'
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 39
    with open("common/logs/telemetry.log", "a") as fl:
IndentationError: expected an indented block after 'with' statement on line 38
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 39
    with open("common/logs/telemetry.log", "a") as fl:
IndentationError: expected an indented block after 'with' statement on line 38
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 39
    with open("common/logs/telemetry.log", "a") as fl:
IndentationError: expected an indented block after 'with' statement on line 38
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 39
    with open("common/logs/telemetry.log", "a") as fl:
IndentationError: expected an indented block after 'with' statement on line 38
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 39
    with open("common/logs/telemetry.log", "a") as fl:
IndentationError: expected an indented block after 'with' statement on line 38
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 29
    pass
IndentationError: expected an indented block after 'try' statement on line 28
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 30
    pass pass
IndentationError: expected an indented block after 'try' statement on line 28
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 41
    pass
IndentationError: expected an indented block after 'with' statement on line 40
  File "/root/projects/Bubbles-payload/common/black_swan_agent/simulation_planner.py", line 46
    pass
IndentationError: expected an indented block after 'with' statement on line 45
